app/mailers/views/hypertension_fast_track_pilot_mailer.erb.html

```diff
@@ -5,7 +5,7 @@
   </head>
   <body>
     <p>
-      <% if @submission.form_json.include? "VAMC_Hypertension_Rapid_Decision_Evidence.pdf" %>
+      <% if @has_rrd_claim_processed %>
```

Comment:
This reads better as:
```suggestion
      <% if @rrd_claim_processed %>
```